Test.C

```diff
@@ -11,108 +11,208 @@
 #define NUM_ROUNDS 5
 
 //Define structures
-struct Equipment {
-    char name[NAME_LENGTH]; // name of the piece of Equipment
+struct Equipment
+{
+    char name[NAME_LENGTH];               // name of the piece of Equipment
     char description[DESCRIPTION_LENGTH]; // short description of Equipment
-    int cost; // cost of the Equipment
-    int messageAmount; // number of messages able to be sent
+    int cost;                             // cost of the Equipment
+    int messageAmount;                    // number of messages able to be sent
 };
 
-struct Encryption {
-    char name[NAME_LENGTH]; // name of the encryption method
+struct Encryption
+{
+    char name[NAME_LENGTH];               // name of the encryption method
     char description[DESCRIPTION_LENGTH]; // short description of how encryption works
-    int cost; // cost of the encryption
-    double effectiveness; // how effective the encryption is (percentage)
-
+    int cost;                             // cost of the encryption
+    double effectiveness;                 // how effective the encryption is (percentage)
 };
 
-struct Attack {
-    char name[NAME_LENGTH]; // name of the attack
+struct Attack
+{
+    char name[NAME_LENGTH];               // name of the attack
     char description[DESCRIPTION_LENGTH]; // short description of how the attack works
-    int tier; // level of intensity
+    int tier;                             // level of intensity
 };
 
-struct EncryptionInventory {
-    Encryption encryption; // the data
+struct EncryptionInventory
+{
+    Encryption encryption;     // the data
     EncryptionInventory *next; // the pointer
 };
 
-struct Player {
-    char name[NAME_LENGTH]; // the player's name
+struct Player
+{
+    char name[NAME_LENGTH];                      // the player's name
     int equipmentInventory[NUM_EQUIPMENT] = {0}; // the current equipment the user owns
-    EncryptionInventory encryptionInventory; // the current encryption methods the user owns
-    int currentCredits; // the current number of credits the user has available
+    EncryptionInventory encryptionInventory;     // the current encryption methods the user owns
+    int currentCredits;                          // the current number of credits the user has available
 };
 
-struct Round {
+struct Round
+{
     int attackTier;
     Attack attack;
 };
 
 //Function prototypes
 void displayBriefingMessage(Player *playerPtr);
 
-int main(){
+void equipmentStore(struct Player *playerPtr)
+{
+    int equipmentSelect;
+    int amountSelect;
+    int transactionTotal;
+    char yesNo;
+
+    printf("|------------------------------ Equipment  Store ------------------------------|\n");
+
+    for(int x = 0; x < NUM_EQUIPMENT; x++){ 
+        printf("%s %s %d %d\n", equipmentStock[x][0] equipmentStock[x][1] equiptmentStock[x][2] equiptmentStock[x][3]);
+    }
```

Comment:
Stick to the standard index variables, `i` then `j` then `k`. `x` is often used for things like position on a screen.

`equipmentStock` is not a 2D array, and therefore cannot be accessed with double brackets `[i][0]`. Cherish had this same issue. Remember, you are referencing attributes of a structure, treat it like an object with OOP.

And don't forget your commas between function parameters.
```suggestion
    int i;
    for(i=0; i<NUM_EQUIPMENT; i++){ 
        printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equiptmentStock[i].cost, equiptmentStock[i].messageAmount);
    }
```